ehrapy/preprocessing/_encode.py

```diff
@@ -103,6 +106,10 @@ def encode(
     else:
         print(f"[b red]Cannot encode object of type {type(data)}. Can only encode AnnData or MuData objects!")
         raise ValueError
+    if isinstance(data, AnnData):
+        logg.info("Encoded the AnnData object.")
```

Comment:
Don't think that we need these logging messages as INFO. They don't provide any info. They can be DEBUG level